packages/strapi-plugin-upload/admin/src/components/FiltersCard/Wrapper.js

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div``;
+
+export default Wrapper;
```

Comment:
is it normal ?

Reply:
Yes, because the filters compos will be updated in this draft PR https://github.com/strapi/strapi/pull/5333